app/models/concerns/operations/stock_operations.rb

```diff
@@ -114,7 +114,7 @@ def register_loss(package, quantity:, location: nil, action: 'loss', description
     # @param [String] action: the inventory action (optional)
     # @param [String] description: notes to detail the action
     #
-    def register_quantity_change(package, quantity:, location:, action:, description: nil, source: nil)
+    def register_quantity_change(package, quantity:, location:, action:, description: nil, source: nil, processing_destination_lookup_id: nil)
```

Comment:
That parameter doesn't have its place here. Please remove that. 
The processing destination _is_ a source, which is already in the params.

The _caller_ of this method should pass in the source